apps/connect/src/components/ConnectLoader.tsx

```diff
@@ -21,11 +21,17 @@ export default function ConnectLoader({ config }: WormholeLoaderProps) {
   useEffect(() => {
     const script = document.createElement("script");
     script.src = `assets/wormhole-connect/main.js`;
+    if (import.meta.env.VITE_APP_JS_WC_INTEGRITY_SHA_384) {
+      script.integrity = `sha384-${import.meta.env.VITE_APP_JS_WC_INTEGRITY_SHA_384}`;
```

Comment:
This is amazing! I was not aware of this feature ([docs](https://developer.mozilla.org/en-US/docs/Web/Security/Subresource_Integrity))